packages/app/src/lib/sanity.tsx

```diff
@@ -109,15 +109,18 @@ export function getImageProps<T extends ImageBlock>(
   const { metadata } = asset;
 
   const {
-    defaultWidth = node.asset.metadata.dimensions.width,
+    defaultWidth = metadata.dimensions.width,
     sizes: sizesOption,
   } = options;
 
   const width = findClosestSize(defaultWidth, imageResizeTargets);
   const height = width / metadata.dimensions.aspectRatio;
 
   const src = getImageSrc(node.asset, defaultWidth);
-  let srcSet = undefined; //we keep this undefined for SVG's, which don't need srcSets
+
+  /* we keep these undefined for SVGs, which don't need srcSets */
+  let srcSet = undefined;
+  let srcSetWebp = undefined;
```

Comment:
I don't think assigning undefined is any different from assigning nothing. But since these can also become strings I think this would be more correct?

```suggestion
  let srcSet: undefined | string;
  let srcSetWebp: undefined | string;
```